help/v3.9.1/index.html

```diff
@@ -0,0 +1,74 @@
+                <li><a href ="configuration/general_settings.html" target="myPage">Transformation and utlity model.</a></li>
+                <li><a href ="configuration/research_subset.html"  target="myPage">Extracting a research sample.</a></li>
+                <li><a href ="global_settings.html" target="myPage">Project settings.</a></li>
+
+            </ol>
+            </li>
+            <li><a href ="exploration/overview.html" target="myPage">Exploration.</a>
+            <ol>
+                <li><a href ="exploration/solution_space.html" target="myPage">Visualizations of the solution space.</a></li>
+                <li><a href ="exploration/filtering.html" target="myPage">Filtering the space.</a></li>
+                <li><a href ="exploration/clipboard.html" target="myPage">Using the clipboard</a></li>
+                <li><a href ="exploration/properties.html" target="myPage">Properties of transformations</a></li>
+            </ol>
+            </li>
+            <li><a href ="utility/overview.html" target="myPage">Utility analysis and enhancement.</a>
+            <ol>
+                <li><a href ="utility/data.html" target="myPage">Comparing data.</a></li>
+                <li><a href ="utility/summary.html" target="myPage">Summary statistics</a></li>
+                <li><a href ="utility/distribution.html" target="myPage">Empirical distribution</a></li>
+                <li><a href ="utility/contingency.html" target="myPage">Contingency table</a></li>
+                <li><a href ="utility/classes.html" target="myPage">Classes and records</a></li>
+                <li><a href ="utility/inputproperties.html" target="myPage">Input properties</a></li>
+                <li><a href ="utility/outputproperties.html" target="myPage">Output properties</a></li>
+                <li><a href ="utility/accuracy.html" target="myPage">Classification accuracy</a></li>
+                <li><a href ="utility/quality.html" target="myPage">Quality models</a></li>
+                <li><a href ="utility/localrecoding.html" target="myPage">Local recording</a></li>
+
+            </ol>
+            </li>
+            <li><a href ="risk/overview.html" target="myPage">Risk analysis.</a>
+            <ol>
+                <li><a href ="risk/classsizes.html" target="myPage">Distribution of risks</a></li>
+                <li><a href ="risk/quasiidentifiers.html" target="myPage">Quasai-identifiers</a></li>
+                <li><a href ="risk/reidentification.html" target="myPage">Re-identification risks</a></li>
+                <li><a href ="risk/hipaa.html" target="myPage">HIPAA identifiers</a></li>
+                <li><a href ="risk/uniques.html" target="myPage">Population uniques</a></li>
+                <li><a href ="risk/population.html" target="myPage">Population settings</a></li>
+            </ol>
+            </li>
+        </ol>
+        <iframe name="myPage" id="myPage" width="800px" height="999px"   frameborder="0" allowfullscreen>
```

Comment:
Why the inframe? I would prefer to have it removed and target _self or _blank being used in the links.